src/pages/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -78,6 +78,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 - There's a new method, **Send Thinking Indicator** ([Web](/messaging/agent-chat-api/#send-thinking-indicator) & [RTM](/messaging/agent-chat-api/rtm-reference/#send-thinking-indicator)).
 - There's a new push, [**incoming_thinking_indicator**](/messaging/agent-chat-api/rtm-pushes/#incoming_thinking_indicator).
+- New thread summary feature is introduced. It is available for `business` and `enterprise` licenses with `chats.auto_summary_enabled` license property set to `true` (default). 
```

Comment:
```suggestion
```

Reply:
The `chats.auto_summary_enabled` property is configurable through configuration-api. If possible, I would leave this bit, so that there's some mention of how to disable this feature with API in the docs. It was requested to include it in the ticket as well: https://livechatinc.atlassian.net/browse/ME-883

I'll remove information about the required plan though